internal/util/http.go

```diff
@@ -36,7 +36,9 @@ type HttpClient struct {
 func NewHttpClient(logger *log.Logger) HttpClient {
 	return HttpClient{
 		client: &http.Client{
-			Timeout: 10 * time.Second,
+			Transport: &http.Transport{
```

Comment:
From the net/http package,

> // If nil, DefaultTransport is used.
        Transport RoundTripper

So 524677a replaces the DefaultTransport.